### PR TITLE
Book component 🕺🏿

### PR DESCRIPTION
**Pull request by moralesbang**
### Summary

- Added book-card component.
- Removed code from principal view created by template.

### Screenshots

Screen Shot 2020-05-18 at 4 16 10 PM

### Trello Card

https://trello.com/c/EvuS2Ypo

### CODE REVIEW

File: src/app/components/book-card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  title: string;
+  author?: string;
```
[the remark this answers is not in the document]
Comment (moralesbang) [reply src/app/components/book-card/index.tsx:8]: That was a test that I forgot to remove 😁

File: src/app/components/book-card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  coverImageURI: string;
+}
+
+function BookCard({ title, author = 'Anonymous', coverImageURI }: Props) {
```
[the remark this answers is not in the document]
Comment (moralesbang) [reply src/app/components/book-card/index.tsx:12]: Forgot that. I'm gonna remove it.

File: src/app/components/book-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+
+import styles from './styles';
+
```
[the remark this answers is not in the document]
Comment (moralesbang) [reply src/app/components/book-card/index.tsx:5]: Cool

File: src/app/index.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { SafeAreaView } from 'react-native';
+
+import BookList from './views/book-list';
```
[the remark this answers is not in the document]
Comment (moralesbang) [reply src/app/index.tsx:4]: Vue bia :V

**Comment by moralesbang** [comment]
@sfernandez11 This is just the book card's component. The complete list gonna be in other PR according to the trello board.

**Comment by sfernandez11** [comment]
@moralesbang I know but for you the book card's component is the same that Invision?

File: src/app/screens/BookList/index.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { View } from 'react-native';
+import BookCard from '@app/components/BookCard';
```
Comment (sfernandez11) [inline src/app/screens/BookList/index.tsx:3]: `import BookCard from '@components/BookCard';`

**Pull request merged**